src/js/components/RangeSelector/__tests__/__snapshots__/RangeSelector-test.js.snap

```diff
@@ -2772,8 +2772,8 @@ exports[`RangeSelector min 1`] = `
   outline: none;
   max-width: 100%;
   margin: 0px;
-  background-color: rgba(253,111,255,0.4);
-  color: #444444;
+  background-color: rgba(125,76,219,0.4);
```

Comment:
I believe this is right now. previous the test snapshots were using the dark control color (accent-1) and I believe it should be the `brand` color since no dark is being set for this theme.